src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -92,18 +92,17 @@ protected function registerBorder($style)
         $styleId = $style->getId();
 
         if ($style->shouldApplyBorder()) {
-
             $border = $style->getBorder();
             $serializedBorder = serialize($border);
 
-            if (!isset($this->registeredBorders[$serializedBorder])) {
-
+            if (isset($this->registeredBorders[$serializedBorder])) {
+                $this->styleIdToBorderMappingTable[$styleId] = $this->registeredBorders[$serializedBorder];
```

Comment:
So `$this->registeredBorders[$serializedBorder]` contains the styleId associated to the border definition. `$this->styleIdToBorderMappingTable` contains a mapping `$styleId => borderId`. It seems like here the value is a styleId and not a borderId. Am I missing something?